### DISCUSSION TITLE
Language

### DISCUSSION THREAD

**Discussion by dnabookofficiaal** [community/community#124681] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

متاسفانه من به زبان انگلیسی مسلط نیستم، امیدوارم افرادی علاقمند ترجمه به این موضوع علاقمند باشند یا از طریق هوش مصنوعی بشه با سرعت ترجمه انجام بشه و در پروژه های منبع باز مورد استفاده قرار بگیره ایده های من ، چون فکر می‌کنم واقعا موثر هست در پیشرفت و سرعت دستاورد به فناوری های مختلف…

**Comment by LiteBrite82** [comment, marked as answer]
Hi @dnabookofficiaal , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
